src/ConstantData.jsx

```diff
@@ -0,0 +1,4 @@
+export const Option1="All Countries";
+export const Option2="CountryDetails";
+export const Option3="FindNeighbours";
+export const Option4="PersonalInformation";
```

Comment:
This should be like this:

export const sidebarItems = {
ALL_COUNTRIES: "All Countries",
COUNTRY_DETAILS:"countryDetails"
// and so on.
}